tutorials/get-started/index.md

```diff
@@ -0,0 +1,1245 @@
+---
+layout: tutorial
+title: Write Your First Flutter App
+permalink: /tutorials/get-started/
+---
+
+This is a guide to creating your first Flutter app. If you know
+how to program, you can complete this tutorial. You don’t need
+previous experience with Dart or mobile programming.
+
+* TOC
+{:toc}
```

Comment:
Is it possible to not show the sub-section titles, many of which are "Problems?", in the TOC? 

![image](https://user-images.githubusercontent.com/348942/36570575-88aec3f8-17e7-11e8-8eea-750c58ddddc1.png)

Reply:
The autogenerated TOC only goes to level 3 headers. So I popped everything up a level.

Reply:
Btw, it doesn't help. The doc doesn't use headers for the steps. This needs a different solution, for after it lands.